docs/reference/content/getting-started/index.md

```diff
@@ -0,0 +1,17 @@
++++
+date = "2015-03-17T15:36:56Z"
+title = "Getting Started"
+[menu.main]
+  identifier = "Getting Started"
+  weight = 30
+  pre = "<i class='fa fa-road'></i>"
++++
+
+## Getting Started
+
+To help you get started quickly on the new driver, follow:
+
+  * [Installation]({{< ref "getting-started/installation-guide.md" >}}), how to install the Scala driver.
+  * [Quick Tour Primer]({{< ref "getting-started/quick-tour-primer.md" >}}),  background information on the driver and its asynchronous API.
+  * [Quick Tour]({{< ref "getting-started/quick-tour.md" >}}), an quick introduction to using the driver and MongoDB. 
```

Comment:
How about:

Quick Tour, an introduction to using the driver for CRUD operations.
Admin Quick Tour, an introduction to using the driver for administrative operations.